src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdNewCaseCreator.java

```diff
@@ -114,6 +115,18 @@ public CreateCaseResult createNewCase(
             ClientServiceErrorResponse errorResponse = serviceResponseParser.parseResponseBody(exception);
             return new CreateCaseResult(errorResponse.warnings, errorResponse.errors);
         // exceptions received from transformation client
+        } catch (ConstraintViolationException exception) {
+            String message = format(
+                "Invalid response received from transformation endpoint. "
+                    + "Service: %s, exception record: %s, violations: %s",
+                configItem.getService(),
+                exceptionRecord.id,
+                exception.getMessage()
+            );
+
+            log.error(message, exception);
+
+            throw new CallbackException(message, exception);
```

Comment:
Throwing the exception is enough. Logging the error additionally will result in two error entries saying the same thing.

Reply:
do we also want to remove same behaviour in all current catch blocks?